src/testing/tassert.h

```diff
@@ -0,0 +1,5 @@
+#include <assert.h>
+
+#define tassert(e) ((void)((e) ? 0 : __assert(#e, __FILE__, __LINE__)))
```

Comment:
I'm actually not familiar with how this would work -- does this expand out to always be `src/testing/tassert.h`, line `3` or does it work as expected?

Reply:
Yup, it totally works. I just used the definition of assert from assert.h (shared on stackoverflow) and removed the abort().

Here is where I got the idea to use assert redefinition and where I got the definition
https://stackoverflow.com/a/270797/10621852